api/service/legacysync/syncing.go

```diff
@@ -1192,6 +1215,11 @@ func (status *syncStatus) Clone() syncStatus {
 	}
 }
 
+func (ss *StateSync) IsInSync() bool {
```

Comment:
Better rename to IsSynchronized.    InSync was confusing people to be "in sync'ing" or "in sync mode", meaning it's still sync'ing and unsynchronized yet.

Reply:
updated!